src/renderer/components/+workloads-overview/overview.tsx

```diff
@@ -57,11 +58,34 @@ export class WorkloadsOverview extends React.Component<Props> {
   }
 
   render() {
+    const items = workloadsOverviewDetailRegistry.getItems().map((item, index) => {
+      return (
+        <item.components.Details key={`workload-overview-${index}`}/>
+      );
+    });
+
     return (
       <div className="WorkloadsOverview flex column gaps">
-        <OverviewStatuses/>
-        {isAllowedResource("events") && <Events compact hideFilters className="box grow"/>}
+        {items}
       </div>
     );
   }
 }
+
+workloadsOverviewDetailRegistry.add([
+  {
+    components: {
+      Details: (props: any) => <OverviewStatuses {...props} />,
+    }
+  },
+  {
+    priority: 5,
```

Comment:
I know that the default priority is `50`, but since the range is technically `[Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY]` (a closed but non-complete range). Would it make sense to have a PR which changes the default value to be `0`?